src/components/facility/AcousticDensityTelemetry.tsx

```
@@ -0,0 +1,342 @@
+                  type="range"
+                  min="0"
+                  max="100"
+                  value={simScore}
+                  onChange={(e) => setSimScore(Number(e.target.value))}
+                  className="w-full h-1 bg-zinc-200 accent-black rounded-lg cursor-pointer"
+                  data-testid="sim-score-slider"
+                />
+              </div>
+
+              <div className="space-y-1">
+                <label className="text-[10px] font-bold uppercase block text-zinc-500">MQTT Topic</label>
```

ESLint

[error] Replace `MQTT·Topic` with `⏎··················MQTT·Topic⏎················`

(prettier/prettier)